index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-param-names': 1,
+    'jsdoc/check-tag-names': 1,
+    'jsdoc/check-types': 1,
+    'jsdoc/newline-after-description': 0,
```

Comment:
To avoid noise I wouldn't list list rules we want to keep `off` (they're off by default, and current convention of this file is that only settings that imply some change are listed)

Reply:
Will fix

Reply:
fixed